perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -268,6 +268,7 @@ sub report_form_ajax : Path('ajax') : Args(0) {
         $top_message ? (top_message => $top_message) : (),
         unresponsive => $c->stash->{unresponsive}->{ALL} || '',
         by_category => \%by_category,
+        preselected => $c->stash->{'preselected_categories'} || {},
```

Comment:
Could this behave the same as top message, subcategories, display_names above - not outputting an empty hash if there's nothing, please.

Reply:
This is done, and the cypress highways test restored to its original state.